.env.prod

```diff
@@ -4,26 +4,55 @@ REACT_APP_ETHEREUM_RPC_URL=https://mainnet.eth.aragon.network/
 REACT_APP_ETHEREUM_WS_URL=wss://main-light.eth.linkpool.io/ws
```

Comment:
Small nit: Can we rename Polygon variables to
`REACT_APP_POLYGON_<VARIABLE_NAME>` The current convention of having polygon as suffix doesn't make it easy to sort things alphabetically later on if we need to. We can also use the same convention for arbitrum or optimism later

Also I think we should keep all polygon env variables together so

All general envs

All Ethereum env variables

All Polygon env variables

This will also make it easier to spot check a list of which assets are deployed on polygon.

Reply:
done